tests/02_zones/08_log_limit

```diff
@@ -0,0 +1,438 @@
+	#
+
+	define lan_devices = { "br-lan" }
+	define lan_subnets = { 10.0.0.0/24, 192.168.26.0/24, 2001:db8:1000::/60, fd63:e2f:f706::/60 }
+
+	define wan_devices = { "pppoe-wan" }
+	define wan_subnets = { 10.11.12.0/24 }
+
+	define guest_devices = { "br-guest" }
+	define guest_subnets = { 10.1.0.0/24, 192.168.27.0/24, 2001:db8:1000::/60, fd63:e2f:f706::/60 }
+
+
+	#
+	# Limits
+	#
+
+	limit lan_log_limit { rate 1/min ; comment "lan log limit"; }
+
+	limit wan_log_limit { rate 2/min ; comment "wan log limit"; }
+
+	limit guest_log_limit { rate 3/min ; comment "guest log limit"; }
+
+
+	#
+	# User includes
+	#
+
+	include "/etc/nftables.d/*.nft"
+
+
+	#
+	# Filter rules
+	#
+
+	chain input {
+		type filter hook input priority filter; policy drop;
+
+		iifname "lo" accept comment "!fw4: Accept traffic from loopback"
+
+		ct state established,related accept comment "!fw4: Allow inbound established and related flows"
+		tcp dport 7 counter log prefix "@rule[6]: " comment "!fw4: @rule[6]"
+		tcp dport 8 counter comment "!fw4: @rule[7]"
+		tcp dport 9 limit rate 5/min log prefix "@rule[10]: "
+		tcp dport 9 counter comment "!fw4: @rule[10]"
+		iifname "br-lan" jump input_lan comment "!fw4: Handle lan IPv4/IPv6 input traffic"
+		iifname "pppoe-wan" jump input_wan comment "!fw4: Handle wan IPv4/IPv6 input traffic"
+		iifname "br-guest" jump input_guest comment "!fw4: Handle guest IPv4/IPv6 input traffic"
+	}
+
+	chain forward {
+		type filter hook forward priority filter; policy drop;
+
+		ct state established,related accept comment "!fw4: Allow forwarded established and related flows"
+		tcp dport 5 limit name "lan_log_limit" log prefix "@rule[4]: "
+		tcp dport 5 counter comment "!fw4: @rule[4]"
+		tcp dport 6 counter comment "!fw4: @rule[5]"
+		iifname "br-lan" jump forward_lan comment "!fw4: Handle lan IPv4/IPv6 forward traffic"
+		iifname "pppoe-wan" jump forward_wan comment "!fw4: Handle wan IPv4/IPv6 forward traffic"
+		iifname "br-guest" jump forward_guest comment "!fw4: Handle guest IPv4/IPv6 forward traffic"
+	}
+
+	chain output {
+		type filter hook output priority filter; policy drop;
+
+		oifname "lo" accept comment "!fw4: Accept traffic towards loopback"
+
+		ct state established,related accept comment "!fw4: Allow outbound established and related flows"
+		oifname "br-lan" jump output_lan comment "!fw4: Handle lan IPv4/IPv6 output traffic"
+		oifname "pppoe-wan" jump output_wan comment "!fw4: Handle wan IPv4/IPv6 output traffic"
+		oifname "br-guest" jump output_guest comment "!fw4: Handle guest IPv4/IPv6 output traffic"
+	}
+
+	chain prerouting {
+		type filter hook prerouting priority filter; policy accept;
+	}
+
+	chain handle_reject {
+		meta l4proto tcp reject with tcp reset comment "!fw4: Reject TCP traffic"
+		reject with icmpx type port-unreachable comment "!fw4: Reject any other traffic"
+	}
+
+	chain input_lan {
+		tcp dport 1 limit name "lan_log_limit" log prefix "@rule[0]: "
+		tcp dport 1 counter comment "!fw4: @rule[0]"
```

Comment:
https://en.m.wikipedia.org/wiki/TCP_Port_Service_Multiplexer

Please avoid super dangerous ports in examples

Reply:
I will all 1000 to all ports.